src/libostree/ostree-repo-pull.c

```diff
@@ -2131,6 +2133,24 @@ process_one_static_delta_fallback (OtPullData   *pull_data,
   return TRUE;
 }
 
+static void
+enqueue_one_static_delta_part_request_s (OtPullData           *pull_data,
+                                         FetchStaticDeltaData *fetch_data)
+{
+  if (fetcher_queue_is_full (pull_data))
+    {
+      g_debug ("queuing fetch of static delta %s-%s part %u",
+               fetch_data->from_revision ? fetch_data->from_revision : "empty",
```

Comment:
Minor: you can use the GNU C extension `?:` for this: https://gcc.gnu.org/onlinedocs/gcc/Conditionals.html#Conditionals

Reply:
Done. I’ve also added https://github.com/ostreedev/ostree/pull/1599/commits/b4ea90b89449a498d92b405e8629d3175a2b15cc at the beginning of the branch to convert other conditionals in the file too.